cmd/estuary-shuttle/main.go

```diff
@@ -1354,20 +1355,22 @@ func (s *Shuttle) createContent(ctx context.Context, u *User, root cid.Cid, fnam
 
 	resp, err := http.DefaultClient.Do(req)
 	if err != nil {
-		return 0, err
+		return 0, errors.Wrap(err, "failed to Do content create request")
```

Comment:
Best if we just put the "method name" on the error message.

```
"Failed to Do createContent"
```

It's good for log searching too (if we are using `vim` or `nano`).